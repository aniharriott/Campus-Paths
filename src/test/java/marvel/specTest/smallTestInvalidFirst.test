# Find a path between invalid using smallTest.

LoadGraph small smallTest.tsv

FindPath small d a