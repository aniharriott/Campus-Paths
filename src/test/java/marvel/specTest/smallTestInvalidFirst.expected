# Find a path between invalid using smallTest.

loaded graph small

path from d to a:
unknown character d